hedyorg/hedy
# 📜 Remove unnecessary double backticks from translation files

Maybe fixes #5566

Double backticks are displayed to the end user, which is incorrect. The string ``` `{at}``{random}` ``` should be displayed as ``` at random ``` but appears as ``` at``random ```. This PR addresses all occurrences of double backticks.

**How to test**
- Navigate to level 3 and select the Quiz tab. Change the language to Chinese simplified. Start the quiz and ensure that the value in answer 4 does not contain any backticks.
- Navigate to level 3 and select the Dishes adventure. Change the language to Spanish. Check that in the description of the adventure the keywords `borrar de` does not contain backticks.
